src/oas3/operation.ts

```diff
@@ -20,11 +20,11 @@ export function transformOas3Operations(document: OpenAPIObject): IHttpOperation
 
 export const transformOas3Operation: Oas3HttpOperationTransformer = ({ document, path, method }) => {
   const pathObj = maybeResolveLocalRef(document, get(document, ['paths', path])) as PathsObject;
-  if (!pathObj) {
+  if (typeof pathObj !== 'object' || pathObj === null) {
```

Comment:
just curious, `!pathObj` is not enough? Could that variable be a string or?

Reply:
yeah, can be anything